### DISCUSSION TITLE
Synology install

### DISCUSSION THREAD

**Discussion by tpbillund** [blakeblackshear/frigate#1047] [General]
Hi

I'm trying to install Frigate on my Synology NAS DS1618+ inside docker.
Atm I have installed it and can access webinterface, it load... so I must look after the missing config file.
But a question... do I need to install MQTT aswell to get Frigate working, or do I just add my config file to the cameras and then reload the docker and access the webinterface, and then I can see the cameras?

Or will Frigate only work with MQTT, if yes... is there anyone that have a working Synology NAS with the setup running inside Docker?

**Comment by tpbillund** [comment]
I used this, and it works 100% ATM.
`sudo docker run -itd --name=frigate --restart=always --privileged -p xxxMYIPxxx:5005:5000 -p xxxMYIPxxx:1935:1935 --shm-size=1024m --device /dev/dri:/dev/dri -v /dev/bus/usb:/dev/bus/usb -v /volume1/docker/frigate/media:/media/frigate -v /volume1/docker/frigate/config:/config:ro -v /etc/localtime:/etc/localtime:ro -e FONTCONFIG_PATH='/etc/fonts' -e FRIGATE_RTSP_PASSWORD='password' blakeblackshear/frigate:stable-amd64`

And is using MQTT

**Comment by wills106** [comment]
If you want to get Frigate running without using the command line and still have working USB Passthrough take a look at https://github.com/blakeblackshear/frigate/discussions/364

I moved my default Synology port 5000 to another port so Frigate can use 5000.

**Comment by wkethman** [comment]
Command that I used to get Frigate running on Synology with hardware acceleration and Google Coral USB

> sudo docker run -itd --name Frigate --restart=always --privileged -p [Synology IP]:5005:5000 -p [Synology IP]:1935:1935 --shm-size=1024m --mount type=tmpfs,target=/tmp/cache,tmpfs-size=1000000000 --device /dev/dri/renderD128:/dev/dri/renderD128 -v /dev/bus/usb:/dev/bus/usb -v /volume1/docker/frigate/media:/media/frigate -v /volume1/docker/frigate/config:/config:ro -v /etc/localtime:/etc/localtime:ro -e FONTCONFIG_PATH='/etc/fonts' blakeblackshear/frigate:stable-amd64

**Comment by Davidvm86** [comment]
From this issue I found out that I got a permission issue.

https://github.com/blakeblackshear/frigate/issues/1534

I did install frigate as add-on using a synolgy install on the virtual machine manager do you guys have a way to set the permissions right to resolve my issue?

```

This happens when nginx does not have the execute permission on the /media/frigate folder. Can you provide some information on how you have setup storage for HassOS on the synology?
I Have setup the Home assistant in the Virtual manager of the Synology.

in my config.yaml I got following

default_config:
media_source:

I did not change anything else after installing the frigate package and add-on
```
